cookbooks/sinker-with-active-particles/run.sh

```diff
@@ -0,0 +1,7 @@
+# aspect executable 
+ASPECT="/Users/jbnaliboff/Software/aspect/aspect/./aspect"
```

Comment:
Please don't include a hardcoded path like this. Take a look at the other benchmarks.